db/migrate/20210812033033_add_multipart_and_size_columns_to_external_upload_stubs.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+class AddMultipartAndSizeColumnsToExternalUploadStubs < ActiveRecord::Migration[6.1]
+  def up
+    add_column :external_upload_stubs, :multipart, :boolean, default: false, null: false
+    add_column :external_upload_stubs, :external_upload_identifier, :string, null: true, index: true
+    add_column :external_upload_stubs, :filesize, :integer
```

Comment:
Minor but I think it is better if we can specify the units in the column name as well. I'm guessing it is `bytes`. Another thing to note is that if we use an `integer` column, the maximum file size this column can support will be `2147483647` bytes. We probably want to get with a `:bigint` column here insteead?

Reply:
Yes it is bytes; that is a good point with the bigint because it will make the limit 2TB...I am not sure if we will be uploading files bigger than that though? For reference I just used the same type and name as the filesize column in Upload:

https://github.com/discourse/discourse/blob/25c8817e61acad3634e4a6e31088b62a355efcf4/app/models/upload.rb#L516-L516

Reply:
`bigint` should make the limit way more than that since the maximum value it can hold is `9223372036854775807`

Reply:
Oh duh I looked at 2147TB and thought that meant 2TB :joy: I'd much rather it be bytes for consistency, and we should also be changing the `Upload` table as well...that will be a bit of work I assume so I'd prefer to save that for another PR.

Reply:
I think we should discuss this internally where we are already discussing these changes for other tables.